lib/js_rails_routes/generator.rb

```diff
@@ -59,13 +66,22 @@ def handle_route(route_name, route_path)
         keys.push("'#{Regexp.last_match(1)}'")
         route_path.sub!(COMPARE_REGEXP, "' + params.#{Regexp.last_match(1)} + '#{Regexp.last_match(2)}")
       end
-      "export function #{route_name}_path(params) { return process('#{route_path}', params, [#{keys.join(',')}]); }"
+
+      function_name = make_function_name(route_name)
+      "export function #{function_name}(params) { return process('#{route_path}', params, [#{keys.join(',')}]); }"
     end
 
     def routes_with_engines
       @routes_with_engines ||= [default_routes] + subengine_routes
     end
 
+    def make_function_name(route_name)
+      name = route_name
+      name = name.camelize.sub(/^./) { $&.downcase } if lower_camelcase?
+
+      "#{name}#{(lower_camelcase? ? 'Path' : '_path')}"
```

Comment:
Please use `camlize(:lower)` for lower camel case. And we should append `_path` suffix ahead.


```rb
url_helper_name = route_name + '_path'
camelize.nil? ? url_helper_name : url_helper_name.camlize(camlize.to_sym)
```

## ref
http://api.rubyonrails.org/classes/String.html#method-i-camelize